src/handleRequest.ts

```diff
@@ -51,7 +60,8 @@ async function originFetch(req: Request, { acceptEncoding }: OriginFetchOptions,
     const fetchResponse = await fetch(`${origin}${req.url}`, {
         method: req.method,
         headers: convertIncomingHeadersToHeadersInit(headers),
-        body: req.body,
+        body: req.method !== "GET" && req.method != "HEAD" ? Readable.toWeb(req) : undefined,
+        duplex: "half",
```

Comment:
can you elaborate on that a bit more?

Reply:
without that set you get an error saying you have to set it

https://developer.chrome.com/articles/fetch-streaming-requests/#half-duplex